src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1124,6 +1127,16 @@ private IExpressionStatement CreateBoundExpressionStatementOperation(BoundExpres
             return new LazyExpressionStatement(expression, isInvalid, syntax, type, constantValue);
         }
 
+        private ITupleExpression CreateBoundTupleExpressionOperation(BoundTupleExpression boundTupleExpression)
+        {
+            Lazy<ImmutableArray<IOperation>> elements = new Lazy<ImmutableArray<IOperation>>(() => boundTupleExpression.Arguments.SelectAsArray(element => Create(element)));
+            bool isInvalid = boundTupleExpression.HasErrors;
+            SyntaxNode syntax = boundTupleExpression.Syntax;
+            ITypeSymbol type = boundTupleExpression.Type;
+            Optional<object> constantValue = ConvertToOptional(boundTupleExpression.ConstantValue);
```

Comment:
I'd still remove this logic. Tuples don't have constant values. 

---
In reply to: [126766304](https://github.com/dotnet/roslyn/pull/20749#discussion_r126766304) [](ancestors = 126766304)